src/quoridor/QuoridorPlayer.cs

```diff
@@ -0,0 +1,35 @@
+﻿namespace QuoridorEngine.src.quoridor
+{
+    /// <summary>
+    /// Class representing quoridor player
+    /// </summary>
+    internal class QuoridorPlayer
+    {
+        private readonly int totalWalls = 10;
+        private int availableWalls;
+        private int victories = 0;
+        private int defeats = 0;
+        private int draws = 0;
+        private bool isWhite;
+        private int x;
+        private int y;
+
+        public int AvailableWalls { get; }
+        public bool IsWhite { set; get; }
```

Comment:
Limit write access to this variable to the constructor only. We shouldn't be allowing this to be dynamically changing during the course of a game